aviary/examples/run_reserve_mission_multiphase_fixedtime.py

```diff
@@ -0,0 +1,204 @@
+            "polynomial_control_order": 1,
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_distance": False,
+            "initial_mach": (0.2, "unitless"),
+            "final_mach": (0.72, "unitless"),
+            "mach_bounds": ((0.18, 0.74), "unitless"),
+            "initial_altitude": (0.0, "ft"),
+            "final_altitude": (32000.0, "ft"),
+            "altitude_bounds": ((0.0, 34000.0), "ft"),
+            "throttle_enforcement": "path_constraint",
+            "fix_initial": True,
+            "constrain_final": False,
+            "fix_duration": False,
+            "initial_bounds": ((0.0, 0.0), "min"),
+            "duration_bounds": ((64.0, 192.0), "min"),
+        },
+        "initial_guesses": {"times": ([0, 128], "min")},
+    },
+    "cruise": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+        "user_options": {
+            "reserve": False,
+            "optimize_mach": False,
+            "optimize_altitude": False,
+            "polynomial_control_order": 1,
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_distance": False,
+            "initial_mach": (0.72, "unitless"),
+            "final_mach": (0.72, "unitless"),
+            "mach_bounds": ((0.7, 0.74), "unitless"),
+            "initial_altitude": (32000.0, "ft"),
+            "final_altitude": (34000.0, "ft"),
+            "altitude_bounds": ((23000.0, 38000.0), "ft"),
+            "throttle_enforcement": "boundary_constraint",
+            "fix_initial": False,
+            "constrain_final": False,
+            "fix_duration": False,
+            "initial_bounds": ((64.0, 192.0), "min"),
+            "duration_bounds": ((56.5, 169.5), "min"),
+        },
+        "initial_guesses": {"times": ([128, 113], "min")},
+    },
+    "descent": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+        "user_options": {
+            "reserve": False,
+            "optimize_mach": False,
+            "optimize_altitude": False,
+            "polynomial_control_order": 1,
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_distance": False,
+            "initial_mach": (0.72, "unitless"),
+            "final_mach": (0.36, "unitless"),
+            "mach_bounds": ((0.34, 0.74), "unitless"),
+            "initial_altitude": (34000.0, "ft"),
+            "final_altitude": (500.0, "ft"),
+            "altitude_bounds": ((0.0, 38000.0), "ft"),
+            "throttle_enforcement": "path_constraint",
+            "fix_initial": False,
+            "constrain_final": True,
+            "fix_duration": False,
+            "initial_bounds": ((120.5, 361.5), "min"),
+            "duration_bounds": ((29.0, 87.0), "min"),
+        },
+        "initial_guesses": {"times": ([241, 58], "min")},
+    },
+    "post_mission": {
+        "include_landing": False,
+        "target_range": (1906, "nmi"),
+    },
+    "climb_reserve": {
```

Comment:
Just a note that there are a lot of values to fill in to define a reserve mission. How feasible would it be to let the user define a minimal set, say (distance, mach) and have aviary build the phases for it? Note: I am not sure how we define these in FLOPS, so I suspect it is simpler.

Reply:
I think ideally these would be filled in by the phase creator UI to start with, so then if the user went into the phase_info file created by the UI they could have all of these things already set. I do agree it's a lot of items and I don't think we have a sense of "defaults right now". But defaults is more "magic", and "magic" isn't all good.

Reply:
I feel like this is out of scope for the current jog. I like making things easier for people, but the focus is on enabling reserve missions of arbitrary ordering.

Reply:
for anything outside the scope of this jog/PR, could you add an issue?